.docker/scripts/ckan_cli

```diff
@@ -58,10 +58,17 @@ fi
 
 if [ "$COMMAND" = "ckan" ]; then
     echo "Using 'ckan' command from $ENV_DIR with config ${CKAN_INI}..." >&2
-    exec $ENV_DIR/ckan -c ${CKAN_INI} "$@"
+    # adjust args to match ckan expectations
+    COMMAND=$(echo "$1" | sed -e 's/create-test-data/seed/')
```

Comment:
what does this do?

Reply:
It changes the action name from `create-test-data` (paster syntax) to `seed` (ckan syntax).